documentation/src/components/side-nav.ts

```diff
@@ -86,11 +91,15 @@ class SideNav extends SpectrumElement {
 
     render(): TemplateResult {
         return html`
-            <div class="scrim" @click=${this.toggle}></div>
+            <sp-underlay
+                class="scrim"
+                @click=${this.close}
+                ?open=${this.open}
+            ></sp-underlay>
             <aside>
                 <div id="nav-header">
                     <div id="logo-container">
-                        <a href="#">
+                        <a href="#index">
```

Comment:
We've gone back and forth on whether this rule is required. `#` is technically a legit destination, but not _explicit_. Any insight on the accessibility of `#` vs `#index` or `#top` will be submitted back to the linter!